libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/Resolver.kt

```diff
@@ -33,6 +33,7 @@ enum class ResolverKeyKind {
   Schema,
   CustomScalarAdapters,
   Pagination,
+  ArgumentDefinition,
```

Comment:
For later: we can use the coordinate to lookup the matching kotlin instead of adding a new kind each time.